### DISCUSSION TITLE
Tenant User management

### DISCUSSION THREAD

**Discussion by ajsatix** [awslabs/aws-saas-boost#52] [Q&A]
Post tenant onboarding, does SaaS boost allow the tenant admin to manage the tenant users through the user module?

**Comment by brtrvn** [comment]
The user service is for system administration users (employees of the SaaS vendor), not for the users of the tenants (individuals of the SaaS vendor's customer). So, the existing user service manages identities who have access to the SaaS Boost admin web UI.

We intend to provide a tenant user service (identity service) as part of our future roadmap. One of the reasons we did not provide this initially is that it will require integration/modifications of the application and because there are so many ways to approach identity, we wanted to gather community feedback.

**Comment by AffiTheCreator** [comment]
+1 for this feature

**Comment by rshettyami** [comment]
I see that this question was posted in 2021, is the "Tenant User" admin available now?

**Comment by brtrvn** [comment]
This functionality is on the roadmap in the researching phase. Please follow and add feedback to the tracking ticket https://github.com/awslabs/aws-saas-boost/issues/316.
